### PR TITLE
add libcue

### PR DESCRIPTION
**Pull request by koltiradw**

### CODE REVIEW

File: projects/libcue/fuzz_cue_parse_string.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <stdlib.h>
```
Comment (kobrineli) [inline projects/libcue/fuzz_cue_parse_string.c:1]: add license

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/libcue/fuzz_cue_parse_string.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#include <string.h>
+#include <stdint.h>
+
+#include "libcue.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  char *fuzz_data = (char *)malloc(Size + 1);
+  memcpy(fuzz_data, Data, Size);
+  fuzz_data[Size] = '\0';
+  Cd *cd = cue_parse_string (fuzz_data);
```
Comment (apach301) [inline projects/libcue/fuzz_cue_parse_string.c:27]: Extra space

File: projects/libcue/build.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+make -j$(nproc)
+
+$CC $TARGET_CFLAGS -I/libcue /fuzz_cue_parse_string.c libcue.a -o fuzz_cue_parse_string
+
+cd ..
+
+mkdir libfuzzer && cd libfuzzer
+
+CC=clang
+cmake .. -DCMAKE_C_COMPILER=$CC \
```
Comment (apach301) [inline projects/libcue/build.sh:41]: Duplicate `..` here and in following cmake commands

**Review by apach301 — CHANGES_REQUESTED** [review]

**Review by apach301 — APPROVED** [review]

File: projects/libcue/build.sh [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#!/bin/bash -eu
```
Comment (kobrineli) [inline projects/libcue/build.sh:17]: move shebang to the first line

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/libcue/build.sh [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+	..
+make -j$(nproc)
+
+$CC $TARGET_CFLAGS -I/libcue /fuzz_cue_parse_string.c libcue.a -o fuzz_cue_parse_string
+
+cd ..
+
+mkdir cover && cd cover
+
+TARGET_CFLAGS="-fprofile-instr-generate -fcoverage-mapping"
```
Comment (kobrineli) [inline projects/libcue/build.sh:50]: better use `-g` here as well

**Review by kobrineli — CHANGES_REQUESTED** [review]

**Review by kobrineli — APPROVED** [review]

**Pull request merged**